jws/jws.go

```diff
@@ -136,7 +144,7 @@ func Sign(payload JWSPayload, did did.BearerDID, opts ...SignOpts) (string, erro
 		return "", fmt.Errorf("failed to determine alg: %s", err.Error())
 	}
 
-	header := Header{ALG: jwa, KID: verificationMethodID}
+	header := Header{ALG: jwa, KID: verificationMethodID, TYP: o.typ}
```

Comment:
what happens if `o.typ` is empty? would it cause any issues? should there be some default?

Reply:
I _think_ the answer to this is that the json marshaling omits it so it's as if this wasn't there (I tested this so I know it to be true, only expressing doubt b/c of my golang newbieness)